cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TemplateUtils.java

```diff
@@ -51,12 +152,11 @@ public static Map<String, String> annotations(HasMetadataTemplate template)   {
     /**
      * Extracts the deployment strategy configuration from the Deployment template
      *
-     * @param template      Deployment template which maybe contains custom deployment strategy configuration
-     * @param defaultValue  The default value which should be used if the deployment strategy is not set
-     *
-     * @return  Custom deployment strategy or default value if not defined
+     * @param template     Deployment template which maybe contains custom deployment strategy configuration
+     * @param defaultValue The default value which should be used if the deployment strategy is not set
+     * @return Custom deployment strategy or default value if not defined
      */
```

Comment:
No need to change the whitespaces unrelated to the PR. The original formatting is the one we actually prefer.